wordpress-mobile/WordPress-Android
# Remove domains transfer card

Removes domain transfer card on dashboard since campaign is wrapped up. p5uIfZ-dJC-p2#comment-21683

| Domain Transfer Card | Domain Transfer Intro |
|--------|--------|
| ![Screenshot_20230802_133138](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/fde5e739-9a94-4319-bae7-c7cb308e2e1e) |![Screenshot_20230729_145335](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/026d3e4a-33f5-4db3-8c76-776c930dbef8) | 
-----

## To Test:
Test 1
- Launch JP app
- Select a site where you're **admin**, and **not a P2**, and haven't previously **hidden this card**
- `Verify` domain transfer card is not shown as in first image above

Test 2
- `Navigate` to `Domains`
- `Tap` on **ALL** at the top right hand corner on **Site Domain** screen
- `Tap` on **+** on top right hand corner
- `Tap` on **Transfer domain** button at the bottom
- `Verify` it opens Transfer Your Domains in WebView as in second image above

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

3. What automated tests I added (or what prevented me from doing so)

    - Updated unit tests

-----

## PR Submission Checklist:


-----
